sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -93,9 +87,10 @@ export class ServiceBusClient {
       config = ConnectionConfig.create(hostOrConnectionString);
 
       options = credentialOrServiceBusClientOptions as ServiceBusClientOptions;
-      config.webSocket = options && options.webSocket;
+      config.webSocket = options && options.webSocketOptions?.webSocket;
       config.webSocketEndpointPath = "$servicebus/websocket";
-      config.webSocketConstructorOptions = options && options.webSocketConstructorOptions;
+      config.webSocketConstructorOptions =
+        options && options.webSocketOptions?.webSocketConstructorOptions;
```

Comment:
Now that we're using null-ish coalescing, can this be further reduced to:
```suggestion
        options?.webSocketOptions?.webSocketConstructorOptions;
```
Same suggestion applies 2 lines up.

Reply:
done